aten/src/ATen/quantized/Copy.cpp

```diff
@@ -0,0 +1,20 @@
+#include <ATen/native/Copy.h>
+
+#include <ATen/ATen.h>
+#include <ATen/CPUApplyUtils.h>
+
+namespace at { namespace native {
+Tensor _s_copy_from_quantized(
+    const Tensor& self,
+    const Tensor& dst,
+    bool non_blocking) {
+  AT_CHECK(self.scalar_type() == at::kQInt8, "Quantized copy only works with kQInt8 as source Tensor");
+  AT_CHECK(dst.scalar_type() == at::kByte, "Quantized copy only works with kByte as target Tensor");
+  at::CPU_tensor_apply2<uint8_t, qint8>(
```

Comment:
I think just reiterpret_cast + memcpy would do and they would be much faster

Reply:
Dont we still need a copy method for Qtensors? Are we dis-allowing copy for this type of tensor?

Reply:
@raghuramank100 technically this function is not copy. It is used only for int_repr()
But I think we still should allow copy for QTensor? @dzhulgakov